clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -814,6 +814,13 @@ static Expected<StringRef> runCompile(StringRef &InputFile,
 
   SmallVector<StringRef, 8> CmdArgs;
   CmdArgs.push_back(*LLCPath);
+  // Checking for '-shared' linker option
+  for (const opt::Arg *Arg : Args) {
+    if (Arg->getOption().matches(OPT_shared)) {
```

Comment:
Instead of looping through the args, maybe `Args.hasArg(OPT_shared)` would be better

Reply:
Ah.. Good point. Thanks Mike. Will address.

Reply:
Done